domains/pretzel.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "tobaiscrowley"
+        "discord": "gus_polinksi"
+    },
+    "records": {
+        "CNAME": "https://microblog-aa389f.gitlab.io"
```

```suggestion
        "CNAME": "microblog-aa389f.gitlab.io"
```

CNAME records cannot contain protocols (`https://`) or slashes (`/`).